serving/resource_model_serving_test.go

```diff
@@ -0,0 +1,409 @@
+package serving
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/endpoints"
+	"github.com/databricks/terraform-provider-databricks/qa"
+)
+
+func TestModelServingCornerCases(t *testing.T) {
+	qa.ResourceCornerCases(t, ResourceModelServing())
+}
+
+func TestModelServingCreate(t *testing.T) {
+	qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   http.MethodPost,
+				Resource: "/api/2.0/serving-endpoints",
+				ExpectedRequest: endpoints.CreateServingEndpoint{
+					Name: "test-endpoint",
+					Config: endpoints.EndpointCoreConfigInput{
+						ServedModels: []endpoints.ServedModelInput{
+							{
+								Name:               "prod_model",
+								ModelName:          "ads1",
+								ModelVersion:       "2",
+								WorkloadSize:       "Small",
+								ScaleToZeroEnabled: true,
+							},
+							{
+								Name:               "candidate_model",
+								ModelName:          "ads1",
+								ModelVersion:       "4",
+								WorkloadSize:       "Small",
+								ScaleToZeroEnabled: false,
+							},
+						},
+						TrafficConfig: &endpoints.TrafficConfig{
+							Routes: []endpoints.Route{
+								{
+									ServedModelName:   "prod_model",
+									TrafficPercentage: 90,
+								},
+								{
+									ServedModelName:   "candidate_model",
+									TrafficPercentage: 10,
+								},
+							},
+						},
+					},
+				},
+				Response: endpoints.ServingEndpointDetailed{
+					Name: "test-endpoint",
+				},
+			},
+			{
+				Method:   http.MethodGet,
+				Resource: "/api/2.0/serving-endpoints/test-endpoint?",
+				ExpectedRequest: endpoints.GetServingEndpointRequest{
```

Comment:
you need to remove `ExpectedRequest` block here, and do this to all `http.MethodGet`

Reply:
doing that would fix all the `Input ('') needs to be valid json.` error